bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/obscomp/SPInstObsComp.java

```diff
@@ -461,18 +462,22 @@ public Instrument getInstrument() {
     // DHS_WRITE_TIME. The class has a `serialVersionUID` and not-updated
     // clients will be expecting a DHS_WRITE_TIME field. Adding a `readObject`
     // to initialize the value for not-updated clients.
-    private PlannedTime.CategorizedTime DHS_WRITE_TIME;
+    /*
+    This method is not being used. I have commented to check it on the pull-request.
+    private PlannedTime.CategorizedTime WRITE_TIME;
+
 
     private void readObject(java.io.ObjectInputStream in)
         throws IOException, ClassNotFoundException {
 
         in.defaultReadObject();
 
-        DHS_WRITE_TIME = getDhsWriteTime();
+        WRITE_TIME = getWriteTime();
     }
-
-    public final PlannedTime.CategorizedTime getDhsWriteTime() {
-        return DHS_WRITE_TIMES.get(getInstrument());
+    */
+    public final PlannedTime.CategorizedTime getWriteTime() {
+        return WRITE_TIMES.get(getInstrument());
     }
```

Comment:
This one too -- as much as I prefer (and suggested we generalize the write time since it is not all DHS now), we'd need to leave it at `getDhsWriteTime` to avoid serialization issues.